devtools/conda-envs/test_env.yaml

```diff
@@ -1,37 +1,19 @@
 name: openmmforcefields-test
-
 channels:
   - conda-forge
-
 dependencies:
-    # Base depends
   - python
   - pip
-
-    # Testing
   - pytest
   - pytest-cov
   - pytest-xdist
   - pytest-randomly
-
-    # Requirements for converted force field installer
-  - openmm >=7.6.0
-
+  - openmm >=8.1.2,<9
   - openff-toolkit >=0.11
-
-    # Requirements for conversion tools
   - pyyaml
   - ambertools >=22,<24
   - lxml
   - networkx
-
-    # JSON caching of residue templates
   - tinydb
-
-    # Validating URLs
   - validators
-
-    #
-    # Espaloma requirements below
-    #
-  # espaloma >=0.3.1
+  - rdkit
```

Comment:
Since RDKit is not directly imported or used here, I'm removing it

Reply:
agreeded, and I am not sure if the broken rdkit build was yanked but it looks like the new one has landed and is working

Reply:
The broken builds were yanked (https://github.com/conda-forge/admin-requests/pull/1034), no new version yet I don't think